packages/core-web/src/index.js

```diff
@@ -50,6 +50,41 @@ function detectAndApplyFixedHeaderStyles() {
         margin-top: calc(-${headerHeight}px - ${bufferHeight}rem);
         height: calc(${headerHeight}px + ${bufferHeight}rem);
       }`);
+  insertCss(`.nav-menu-open { max-height: calc(100% - ${headerHeight}px); }`);
+}
+
+function addResizeHeaderListener() {
```

Comment:
shouhld we put this under `detectAndApplyFixedHeaderStyles` instead? (it has a guard for no fixed header as well)

Reply:
Thanks! Shifted this into `detectAndApplyFixedHeaderStyles`.